### PR TITLE
Check if process exist before kill

### PR DESCRIPTION
**Pull request by jin-tron**

### CODE REVIEW

File: cmd/btfs/functest.go [hunk 1]
```diff
@@ -44,10 +44,15 @@ func get_functest(btfsBinaryPath string) error {
...
+		_, err := os.FindProcess(int(cmd.Process.Pid))
 		if err != nil {
-			if !strings.Contains(err.Error(), "process already finished") {
-				fmt.Printf("cannot kill process: [%v] \n", err)
+			log.Info("process already finished\n")
+		} else {
+			err := cmd.Process.Kill()
+			if err != nil {
+				if !strings.Contains(err.Error(), "process already finished") {
+					fmt.Printf("cannot kill process: [%v] \n", err)
```
Comment (taiyangc) [inline cmd/btfs/functest.go]: use `log.Error`

File: cmd/btfs/functest.go [hunk 2]
```diff
@@ -100,10 +105,15 @@ func add_functest(btfsBinaryPath, peerId string) error {
...
```
Comment (taiyangc) [inline cmd/btfs/functest.go]: use `log.Error`

**Comment by taiyangc** [comment]
@jin-tron small change request

File: cmd/btfs/functest.go (hunk 2 above)
Comment (jin-tron) [reply cmd/btfs/functest.go]: fixed

File: cmd/btfs/functest.go (hunk 1 above)
Comment (jin-tron) [reply cmd/btfs/functest.go]: fixed

**Pull request merged**